packages/gmp.rb

```diff
@@ -3,36 +3,36 @@
 class Gmp < Package
   description 'GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating-point numbers.'
   homepage 'https://gmplib.org/'
-  version '6.2.0'
+  version '6.2.1'
   license 'LGPL-3+ and GPL-2+'
   compatibility 'all'
-  source_url 'https://gmplib.org/download/gmp/gmp-6.2.0.tar.lz'
-  source_sha256 '3f33f127bcb6b2c3601676cd3281df45824b148cbf688b73c0fc8248793667d9'
+  source_url 'https://gmplib.org/download/gmp/gmp-6.2.1.tar.lz'
+  source_sha256 '2c7f4f0d370801b2849c48c9ef3f59553b5f1d3791d070cffb04599f9fc67b41'
 
-  binary_url ({
-    aarch64: 'https://github.com/chromebrew/binaries/raw/main/armv7l/gmp-6.2.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://github.com/chromebrew/binaries/raw/main/armv7l/gmp-6.2.0-chromeos-armv7l.tar.xz',
-       i686: 'https://github.com/chromebrew/binaries/raw/main/i686/gmp-6.2.0-chromeos-i686.tar.xz',
-     x86_64: 'https://github.com/chromebrew/binaries/raw/main/x86_64/gmp-6.2.0-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://github.com/chromebrew/binaries/raw/main/armv7l/gmp-6.2.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://github.com/chromebrew/binaries/raw/main/armv7l/gmp-6.2.1-chromeos-armv7l.tar.xz',
+       i686: 'https://github.com/chromebrew/binaries/raw/main/i686/gmp-6.2.1-chromeos-i686.tar.xz',
+     x86_64: 'https://github.com/chromebrew/binaries/raw/main/x86_64/gmp-6.2.1-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '2808a85d261fb740f5278be8ea25aca44de63e7512fbbf5a19bf133db5b5c152',
-     armv7l: '2808a85d261fb740f5278be8ea25aca44de63e7512fbbf5a19bf133db5b5c152',
-       i686: '0bc6669b80774effaaad213e4b13c624fc879f62804672beea1244634c02c1c2',
-     x86_64: '13f50bef5b6d30967fcd227dc750d972d5d2c1f471548c23dce5113dc952eeff',
+  binary_sha256({
+    aarch64: 'c9a07f1b3654f087dff322a5a0d5cbe98fba1e7c5b483621702655a0c879c152',
+     armv7l: 'c9a07f1b3654f087dff322a5a0d5cbe98fba1e7c5b483621702655a0c879c152',
+       i686: '2428a10b21a6b6bfaca1d2bd85b5c2adcd05aa3f48b6e9712d96ff84fa525478',
+     x86_64: '50847bd14c11de841c7c74696e2ff4503253bbaacd38e9ffb31814d6ccfb76f3'
   })
 
   def self.build
-    system "sed -i 's,/usr/bin/file,#{CREW_PREFIX}/bin/file,g' configure"
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-maintainer-mode',
-           '--enable-cxx'
+    system 'filefix'
+    system "./configure \
+      #{CREW_OPTIONS} \
+      --disable-maintainer-mode \
+      --enable-cxx"
     system 'make'
   end
 
   def self.check
+    # Run tests on armv7l after install
     system 'make', 'check'
```

Comment:
This will attempt to run tests for all architectures, not just armv7l.

Reply:
Yes. There's no workaround I could find aside from commenting out the check on armv7l, doing the install, and then taking out the hash and running the make check again.

Reply:
(same issue as with mpfr on x86_64)

Reply:
What is the output of the failed tests?  Maybe we need to work on WHY the tests failed instead of trying to bypass?